control-plane/stor-port/src/types/v0/transport/snapshot.rs

```diff
@@ -120,3 +123,47 @@ pub enum ListReplicaSnapshots {
     /// The specific snapshot.
     Snapshot(SnapshotId),
 }
+
+/// The request type to create a snapshot's clone.
+pub type CreateSnapshotClone = SnapshotCloneParameters;
+
+/// Common set of parameters used for snapshot clone creation.
+#[derive(Debug, Clone)]
+pub struct SnapshotCloneParameters {
+    /// Unique identification of the snapshot source.
```

Comment:
souce snapshot? Rather than snapshot source, as it indicates the volume I feel.

Reply:
Done